tests/tracer/test_compat.py

```diff
@@ -121,3 +121,14 @@ def test_reraise(self):
 )
 def test_is_integer(obj, expected):
     assert is_integer(obj) is expected
+
+
+def test_pep562():
+    with pytest.raises(RuntimeError):
+        from tests.pep562_test import deprecated
+
+        print(deprecated)
```

Comment:
```suggestion
```

Just wanted to check to see if this was intentional. The test will raise if the exception isn't raised but i guess the print might be useful in debugging

Reply:
Yes this was intentional with the idea of actually referencing the imported object. I guess I could maybe add a `# noqa` on line 128 instead.